### ISSUE TITLE
Bootstrap attribute "logLevel" should be reported as a warning finding

### ISSUE THREAD

**Issue by flovogt** [SAP/ui5-linter#602]
Hi team,

for following HTML file we should report a warning finding to use `data-sap-ui-log-level` instead of `data-sap-ui-logLevel`. Documented here https://sdk.openui5.org/topic/91f2d03b6f4d1014b6dd926db0e91070.

```html
<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>MyApp</title>
    <script
      id="sap-ui-bootstrap"
      src="../resources/sap-ui-core.js"
      data-sap-ui-theme="sap_horizon"
      data-sap-ui-resource-roots='{
         "myapp": "./",
      }'
      data-sap-ui-on-init="module:luigi/demo/sample2/index"
      data-sap-ui-language="en"
      data-sap-ui-logLevel="none"
      data-sap-ui-async="true"
      data-sap-ui-compat-version="edge"
    ></script>
  </head>
  <body class="sapUiBody">
    <div
      data-sap-ui-component
      data-name="myapp"
      data-id="container"
      data-settings='{"id" : "app"}'
      id="content"
    ></div>
  </body>
</html>
```

Currently no finding is reported at all.
